src/balena-model.ts

```diff
@@ -366,6 +366,8 @@ export interface Release {
 	update_timestamp: DateString;
 	release_version: string | null;
 	contract: {} | null;
+	is_passing_tests: boolean;
+	release_type: 'final' | 'draft';
```

Comment:
this is actually autogenerated this time (using the `Definition` rather than a rule)

Reply:
100%, see https://github.com/balena-io/open-balena-api/pull/556/files#diff-6fbfd1ba14aab28155305cee9b841695255dd947b2f661fb6e375a4de53403e4R253